examples/timestamp.rs

```diff
@@ -0,0 +1,20 @@
+// example image from http://www.gaia-gis.it/spatialite-2.3.1/resources.html
+
+extern crate rexiv2;
+extern crate chrono;
+
+use rexiv2::Metadata;
+use chrono::NaiveDateTime;
+
+fn main() {
+    if let Ok(meta) = Metadata::new_from_path("examples/DSCN0010.jpg") {
+        if let Ok(time) = meta.get_tag_string("Exif.Image.DateTime") {
+            if let Ok(timestamp) = NaiveDateTime::parse_from_str(&time, "%Y:%m:%d %H:%M:%S") {
```

Comment:
It's definitely more practical for actual use, but should we bother bringing in an extra dev dependency for the example?